Error detail from BQ API was discarded by BQ connector

The problem was reported on [StackOverflow](https://stackoverflow.com/questions/57408388/where-to-find-errors-when-writing-to-bigquery-from-dataproc), I reproduced it. The `errors` from BigQuery API was discarded by BQ Connector:

```
Caused by: java.io.IOException: Error during BigQuery job execution:
{"location":"{tempOutputLocation}",
 "message":"Error while reading data, error message: JSON table encountered too many errors,"
           "giving up. Rows: 3; errors: 1. Please look into the errors[] collection for" 
           "more details.",
 "reason":"invalid"}
```

The related code is [here](https://github.com/GoogleCloudPlatform/bigdata-interop/blob/branch-1.9.x/bigquery/src/main/java/com/google/cloud/hadoop/io/bigquery/BigQueryUtils.java#L108). The `errors` referred by the error message can be get from [JobStatus.getErrors()](https://developers.google.com/resources/api-libraries/documentation/bigquery/v2/java/latest/com/google/api/services/bigquery/model/JobStatus.html).

--- a/bigquery/src/main/java/com/google/cloud/hadoop/io/bigquery/BigQueryUtils.java
+++ b/bigquery/src/main/java/com/google/cloud/hadoop/io/bigquery/BigQueryUtils.java
@@ -106,8 +106,10 @@ public class BigQueryUtils {
         if (pollJob.getStatus().getErrorResult() != null) {
           throw new IOException(
               String.format(
-                  "Job %s failed: %s",
-                  jobReference.getJobId(), pollJob.getStatus().getErrorResult()));
+                  "Job %s failed: %s. Errors: %s",
+                  jobReference.getJobId(),
+                  pollJob.getStatus().getErrorResult(),
+                  pollJob.getStatus().getErrors()));
         }
       } else {
         long millisToWait = pollBackOff.nextBackOffMillis();

